sources/models/shared-defaults/kubernetes-services.toml

```diff
@@ -36,6 +37,10 @@ template-path = "/usr/share/templates/kubernetes-ca-crt"
 path = "/etc/systemd/system/kubelet.service.d/exec-start.conf"
 template-path = "/usr/share/templates/kubelet-exec-start-conf"
 
+[credential-provider-config.yaml]
```

Comment:
You are missing the `configuration-files` prefix, and the `.` will conflict with `toml`'s syntax, instead you should use `-` as in other configurations.